Content.Shared/Silicons/StationAi/SharedStationAiSystem.Held.cs

```diff
@@ -107,42 +108,61 @@ private void OnMessageAttempt(BoundUserInterfaceMessageAttempt ev)
         if (ev.Actor == ev.Target)
             return;
 
+        StationAiWhitelistComponent? whitelistComponent = null;
         if (TryComp(ev.Actor, out StationAiHeldComponent? aiComp) &&
            (!ValidateAi((ev.Actor, aiComp)) ||
-            !HasComp<StationAiWhitelistComponent>(ev.Target)))
+            !TryComp(ev.Target, out whitelistComponent)))
         {
             ev.Cancel();
         }
+
+        if (whitelistComponent is { Enabled: false })
+        {
+            ev.Cancel();
+            _popup.PopupClient(Loc.GetString("ai-device-not-responding"), ev.Actor, PopupType.MediumCaution);
+        }
     }
 
     private void OnHeldInteraction(Entity<StationAiHeldComponent> ent, ref InteractionAttemptEvent args)
     {
         // Cancel if it's not us or something with a whitelist.
         args.Cancelled = ent.Owner != args.Target &&
                          args.Target != null &&
-                         (!TryComp(args.Target, out StationAiWhitelistComponent? whitelist) || !whitelist.Enabled);
```

Comment:
missing Enabled check here actually is hurting by not disabling interaction on simple door opening. Maybe its ok to have check and play popup here